shortfin/tests/apps/llm/components/kvcache/base_attention_cache_test.py

```diff
@@ -539,6 +539,7 @@ def test_free_pages_use_ref_count(
 )
 # fmt: on
 @pytest.mark.asyncio
+@pytest.mark.xfail(reason="xfailed for flakiness on the new-pages-should-be-all-1s assertion. See #1176")
```

Comment:
Should probably have full link to know that it's shark-ai#1176.

Reply:
Good point! We do work with multiple repos. Adding now.